scripts/save.sh

```diff
@@ -108,6 +109,13 @@ pane_full_command() {
 	$strategy_file "$pane_pid"
 }
 
+capture_pane_contents() {
+	local pane_id="$1"
+	local start_line="0"
+	[[ "$(get_tmux_option "$pane_contents_area_option" "visible")" == "full" ]] && start_line="-$2"
+	tmux capture-pane -ep -S "$start_line" -t "$pane_id" > "$(resurrect_pane_file "$pane_id")"
```

Comment:
I hadn't even noticed this option as it doesn't appear in the options summary in my tmux manual page (it's only mentioned in the text description for `capture-pane`).
It does look like a good addition indeed. I just gave it a shot and it seems to be working, so I'll commit it.